src/index.js

```diff
@@ -5,6 +5,7 @@ export * from './icons/index.js';
 export * from './spinner/index.js';
 export * from './alerts/index.js';
 export * from './dropdown-menu/index.js';
+export * from './dropdown-multi-select/index.js';
 export * from './split-pane/index.js';
 export * from './searchbar/index.js';
 export * from './responsive-navbar/index.js';
```

Comment:
We could have a 'dropdown-view' for the dropdown container so that it would be more visible as a separate component. The container could be filled with just div with a text: 'CONTENT' and some background.